reference/twig-extensions/functions/get_media_url.rst

```diff
@@ -0,0 +1,25 @@
+``get_media_url``
+================
```

Comment:
travis said that this line is to short (-:

https://travis-ci.org/sulu-io/sulu-docs/builds/64379442#L106